src/jobs/jobs/bsd_assigned_calls_ttl.js

```diff
@@ -0,0 +1,72 @@
+var pg = require('pg');
+
+var connString = process.env.DATABASE_URL;
+var client;
+
+module.exports = {};
+
+// The default callback allows us to log errors to the console in the event our
+// function is called syncronously.
+// This is prefered over throwing since compilers don't optimize functions
+// containing throw statements.
+var default_cb = module.exports.default_callback = function default_cb(e) {
+  if(e) {
+    console.error(e.stack);
+  }
+};
+
+// Query the database
+module.exports.job = function bsd_assigned_calls_ttl_job(cb) {
+  // If this function wasn't supplied a callback, use the default
+  cb = cb || default_cb
+
+  // Make sure the client exists
+  if(!client) module.exports.init(function fallback_init(e) {
```

Comment:
Why do the separate init and job methods anyway instead of chaining the whole thing into one callback structure?  That way you don't need to do this check.